src/runner.rs

```diff
@@ -24,22 +25,23 @@ impl<'a> Runner<'a> {
         Runner { config }
     }
 
-    pub fn run(&mut self, mut workers: &mut Vec<Box<Worker>>) {
+    pub fn run(&mut self, mut workers: &mut Vec<Box<Worker>>) -> Result<(), FromanError> {
```

Comment:
I'm not sure if `Result<(), ...>` is the "right" way to signal that I don't have a return value. I only care about `Ok` and `Err(something)`.

Reply:
`()` is correct